diff --git a/Cargo.toml b/Cargo.toml
index be9382f4f..d52a18fdd 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -608,3 +608,5 @@ opt-level = 3
 opt-level = 3
 [profile.release.package.twox-hash]
 opt-level = 3
+[patch.crates-io]
+v8 = { path = "../rusty_v8" }
\ No newline at end of file
diff --git a/libs/core/uv_compat/tty.rs b/libs/core/uv_compat/tty.rs
index 5e2dc40c9..870967fa5 100644
--- a/libs/core/uv_compat/tty.rs
+++ b/libs/core/uv_compat/tty.rs
@@ -381,7 +381,7 @@ mod global_termios {
   }
 
   /// Get a pointer to the thread-local errno value.
-  #[cfg(target_os = "macos")]
+  #[cfg(any(target_os = "ios", target_os = "macos"))]
   fn errno_location() -> *mut c_int {
     unsafe extern "C" {
       fn __error() -> *mut c_int;
@@ -397,7 +397,7 @@ mod global_termios {
     unsafe { __errno_location() }
   }
 
-  #[cfg(not(any(target_os = "macos", target_os = "linux")))]
+  #[cfg(not(any(target_os = "ios", target_os = "macos", target_os = "linux")))]
   fn errno_location() -> *mut c_int {
     compile_error!(
       "errno_location not implemented for this platform — \
